config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/FullBundleCreator.java

```diff
@@ -119,8 +119,15 @@ public boolean test(Map<String, String> dependentBundleMap) {
                             dependentBundleMap.get("version").equals(projectInfo.getVersion());
                 }
             });
-
-            jsonFileUtils.createBundleMetadataFile(bundleMetadata, bundleEnvironmentValues.getLeft(), new File(bundleFolderPath));
+            bundleMetadata.put("version", projectInfo.getVersion() + PREFIX_FULL);
```

Comment:
If the project version is not configured, this will be set as `-full`

Reply:
Done